.buildkite/jobs/pipeline.ios_demo_app_rn_66.yml

```diff
@@ -0,0 +1,9 @@
+  - label: ":ios::react: RN.66 + iOS: Demo app"
+    command:
+      - "nvm install"
+      - "JAVA_HOME=/usr/local/opt/openjdk@11/ ./scripts/demo-projects.ios.sh"
```

Comment:
Just wondering, why do we need that `JAVA_HOME` override for iOS?

Reply:
You're right, we don't. Missed it in the copy-paste from the android script 🤭